pyproject.toml

```diff
@@ -1,16 +1,120 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "datatrove"
+version = "1.0.1.dev0"  # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
+description = "HuggingFace library to process and filter large amounts of webdata"
```

Comment:
is it not possible to also include the `long_description` (the readme) as before?

Reply:
The pyproject [spec](https://packaging.python.org/en/latest/specifications/pyproject-toml/) follows PEP621 and does not include a `long_description` field, but the readme can be included and specified in the `readme` key of the `project` table. Should be line 9 in this doc.